Library/Homebrew/keg_relocate.rb

```diff
@@ -45,6 +48,17 @@ def replace_text(text)
       end
       any_changed
     end
+
+    sig { params(value: T.any(String, Regexp)).returns(Regexp) }
+    def self.path_regex(value)
+      value = case value
+      when String
+        Regexp.escape(value)
+      when Regexp
+        value.source
```

Comment:
I assume this is already escaped?

Reply:
The assumption is that if a regex is passed, it shouldn't need to be escaped. If I wanted to match `/foo.bar/` and passed it to this function, I would want the resulting regex to be `/(?<![a-zA-Z0-9])foo.bar/` not `/(?<![a-zA-Z0-9])foo\.bar/`.